cirq/ops/pauli_string.py

```diff
@@ -259,26 +259,6 @@ def keys(self) -> KeysView[raw_types.Qid]:
     def qubits(self) -> Tuple[raw_types.Qid, ...]:
         return tuple(sorted(self.keys()))
 
-    def _circuit_diagram_info_(self, args: 'cirq.CircuitDiagramInfoArgs'
```

Comment:
I'm not sure why we removed the PauliString diagram. Now we have

```
0: ───X───-X───X(0)*Z(2)───1j*X(0)*Y(1)───────────(0.7071067811865476+0.7071067811865475j)*X(0)*Y(1)───
               │           │                      │
1: ────────────┼───────────#2─────────────-1j*Y───#2───────────────────────────────────────────────────
               │
2: ────────────#2──────────────────────────────────────────────────────────────────────────────────────

```

instead of 

```
0: ───PauliString(+X)───PauliString(-X)───PauliString(+X)───PauliString(iX)──────────────────────PauliString((0.707+0.707i)*X)───
                                          │                 │                                    │
1: ───────────────────────────────────────┼─────────────────Y─────────────────PauliString(-iY)───Y───────────────────────────────
                                          │
2: ───────────────────────────────────────Z──────────────────────────────────────────────────────────────────────────────────────
```

is this intentional?